neo/rawio/axonarawio.py

```diff
@@ -498,7 +505,7 @@ def _get_channel_from_tetrode(self, tetrode):
         This function will take the tetrode number and return the Axona
         channel numbers, i.e. Tetrode 1 = Ch1-Ch4, Tetrode 2 = Ch5-Ch8, etc.
         """
-        return np.arange(1, 5) + 4 * (int(tetrode) - 1)
+        return np.arange(0, 4) + 4 * (int(tetrode) - 1)
```

Comment:
This looks like  you are switching to`Tetrode 1 = Ch0-Ch3`, then we should also update the docstring above.

Reply:
Good catch!